app/src/behavior_queue.c

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2022 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#include <zmk/behavior_queue.h>
+
+#include <kernel.h>
+#include <logging/log.h>
+#include <drivers/behavior.h>
+
+LOG_MODULE_DECLARE(zmk, CONFIG_ZMK_LOG_LEVEL);
+
+struct q_item {
+    uint32_t position;
+    struct zmk_behavior_binding binding;
+    bool press;
+    uint32_t wait;
```

Comment:
The compiler is probably going to pad `press` out to 32 bits (https://godbolt.org/z/PnhMazzhq). If you want to be really space-efficient, you could give `press` one bit and `wait` the remaining 31 to pack them both into 32-bits (https://godbolt.org/z/44EevY8a8).

Reply:
Thanks! Fixed.